### ISSUE TITLE
is it possible to access the underlying model via APIs, for example using the ollama APIs?

### ISSUE THREAD

**Issue by msartiano** [ivanfioravanti/chatbot-ollama#60]
Question: is it possible to access the underlying model via APIs, for example using the ollama APIs?

Thanks!

**Comment by hassan4702** [comment]
Yes, it is possible to access underlying AI models through APIs, such as those provided by Ollama. Ollama offers an API that allows developers to integrate and interact with large language models (LLMs) within their applications. 
To access Ollama APIs, you typically need to:

- Set up an API key from the Ollama platform.
- Use the provided endpoints for interacting with the model.
- Send API requests in a defined format (e.g., providing input text) and process the returned model output (e.g., summaries, completions, etc.).

[https://github.com/ollama/ollama](url)
More info in README.md

**Comment by tborchmann** [comment]
If the author would expose the 11434 port this should be possible

**Comment by ivanfioravanti** [comment]
It's better relying on Ollama directly, because it can be hosted on a separate machine from the server hosting chatbot-ollama
